tools/mob_suite/mob_recon.xml

```diff
@@ -0,0 +1,85 @@
+<tool id="mob_recon" name="MOB-Recon" version="1.4.1">
+  <description>(Type contigs and extract plasmid sequences)</description>
+  <requirements>
+     <requirement type="package" version="1.4.1">mob_suite</requirement>
+  </requirements>   
+  <command detect_errors="exit_code">
+    ln -s ${input} ${input.name};
+    mob_recon  --num_threads 16 --infile "${input.name}" --evalue ${adv_param.evalue}  --unicycler_contigs ${adv_param.unicycler_contigs} --run_circlator ${adv_param.run_circlator} --run_typer --outdir ".";
```

Comment:
Don't want to hard-code `-num_threads 16`, have a look at how `${GALAXY_SLOTS:-1}` works.